source/extensions/filters/network/redis_proxy/config.cc

```diff
@@ -47,11 +47,6 @@ Network::FilterFactoryCb RedisProxyFilterConfigFactory::createFilterFactoryFromP
   envoy::extensions::filters::network::redis_proxy::v3::RedisProxy::PrefixRoutes prefix_routes(
       proto_config.prefix_routes());
 
-  // Set the catch-all route from the deprecated cluster and settings parameters.
-  if (prefix_routes.routes_size() == 0 && !prefix_routes.has_catch_all_route()) {
```

Comment:
What does this have to do with the deprecated field?

Reply:
Humm :( My Bad. code pushed accidently. Corrected it.